FileContentOrdering.md

```diff
@@ -62,3 +62,22 @@ internal func updateUnalliedHouses(_ houses: [House]) {
 private maximumHouseSize: Int = 50
 ```
 
+## Convention - Grouping
```

Comment:
Do we want to go a step further and say that if you inherit from different things or conform to different protocols they should all be in separate // MARK: blocks? With a bad example of a UITableViewDelegate and UITableViewDataSource method in the same block?

Related: a common pattern is using an extension in the same file fort he different protocol conforms. Do we want to explicitly address whether we should use // MARK: instead of that?

Reply:
Yeah, I think we should make that clear about different conformances. 

And yeah, I think we can specify that about `// MARK:`

Reply:
Adding more guidance on different conformances and more examples.
As for the explicit extension vs // MARK: I think it's a preference. If there's already an explicit extension then the extension keyword is picked up by Xcode automatically and no explicit // MARK: is necessary. I don't see a particular reason to choose extension vs // MARK: